packages/twenty-front/src/modules/object-record/record-group/hooks/useCurrentRecordGroupDefinition.ts

```diff
@@ -1,37 +1,14 @@
 import { RecordGroupContext } from '@/object-record/record-group/states/context/RecordGroupContext';
-import { hasRecordGroupDefinitionsComponentSelector } from '@/object-record/record-group/states/hasRecordGroupDefinitionsComponentSelector';
-import { recordGroupDefinitionsComponentState } from '@/object-record/record-group/states/recordGroupDefinitionsComponentState';
-import { useRecoilComponentValueV2 } from '@/ui/utilities/state/component-state/hooks/useRecoilComponentValueV2';
-import { useContext, useMemo } from 'react';
+import { recordGroupDefinitionFamilyState } from '@/object-record/record-group/states/recordGroupDefinitionFamilyState';
+import { useContext } from 'react';
+import { useRecoilValue } from 'recoil';
 
-export const useCurrentRecordGroupDefinition = (recordTableId?: string) => {
+export const useCurrentRecordGroupDefinition = () => {
```

Comment:
why "Current here"

Reply:
It's because we're getting the RecordGroup based on the current recordGroupId present in the context, but we can change the naming if you want